src/translations/en.json

```diff
@@ -149,6 +141,8 @@
     "nominalVoltage2": "Nominal Voltage 2",
     "nominalVoltage3": "Nominal Voltage 3",
     "cannotPasteTextAsNominalVoltage": "The clipboard text cannot be set as a positive number :",
+    "cannotRetrieveContingencyList": "Could not retrieve contingency list :",
+    "cannotRetrievefilter": "Could not retrieve filter :",
```

Comment:
in English there is never a space before the ":" character and always a space after
"cannotRetrieveContingencyList": "Could not retrieve contingency list: ",
"cannotRetrievefilter": "Could not retrieve filter: ",

Reply:
ok, I will change